app/scripts/lib/stores/indexeddb-store.test.ts

```diff
@@ -12,111 +12,156 @@ describe('IndexedDBStore', () => {
       const req = indexedDB.deleteDatabase(dbName);
       req.onsuccess = () => resolve();
       req.onerror = () => reject(req.error);
+      req.onblocked = () => {
+        throw new Error("this shouldn't happen. You have an error somewhere");
```

Comment:
I was stumped for a long time because tests were timing out. Having this here would have helped tremendously!

Reply:
haha, this is just in the testing file. but yeah, that would be a funny one.